ERCS/erc-6900.md

```diff
@@ -491,17 +485,13 @@ During validation uninstallation, the account MUST correctly clear flags and oth
 - The account SHOULD call `onUninstall` on the validation module to clean up state if specified by the user.
 - The account MUST emit `ValidationUninstalled` as defined in the interface for all uninstalled validation functions.
 
-### Execution Functions and Their Installation/Uninstallation
-
-- An account can install any number of execution functions.
```

Comment:
Do we also need to keep this?
* An account can install any number of execution functions

Reply:
I thought this was implicitly true from the interfaces and the installation flow - as long as there isn't a selector collision, you can keep installing new execution functions.